static/src/javascripts/projects/common/modules/ui/autoupdate.js

```diff
@@ -164,8 +165,13 @@ define([
                     }
                 }
             }).always(function () {
-                updateDelay(currentUpdateDelay);
-                updateTimeoutId = setTimeout(checkForUpdates, currentUpdateDelay);
+                if (count == 0 || currentUpdateDelay > 0) {
+                    updateDelay(currentUpdateDelay);
+                    updateTimeoutId = setTimeout(checkForUpdates, currentUpdateDelay);
+                } else {
+                    // might have been cached so check straight away
+                    updateTimeoutId = setTimeout(checkForUpdates, 1);
+                }
```

Comment:
I know this is not related to this PR specifically but I usually found all this states management (ifs, state vars) quite error prone. Not much you can do at this stage but to keep in mind if the liveblog logic keep growing.